### ISSUE TITLE
The name for redirect_before_fork option

### ISSUE THREAD

**Issue by wayan** [symkat/Daemon-Control#72]
This is only a question not a real issue. Why the option is called redirect_before_fork? Looking into  the source code I see that the filehandles are redirected **after** all forks are done, just before launching the program. Am I missing something?

**Comment by ikruglov** [comment]
no, you're right. The redirection is done after double-fork. I think it's a bad naming or some historical reasons.
